site/src/api/queries/experiments.ts

```diff
@@ -19,3 +19,7 @@ export const availableExperiments = () => {
 		queryFn: async () => API.getAvailableExperiments(),
 	};
 };
+
+export const isKnownExperiment = (experiment: string): boolean => {
+	return Experiments.includes(experiment as Experiment);
+};
```

Comment:
Alternatively, we could have the FE return the list of known experiments, but it seems silly to do this when we can just use the auto-generated type.